app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 $(function () {
   $('a.preview').attr("target","_blank");
   $('form.preview').attr("target","_blank");
-  $(".select2").select2();
+  $(".select2").select2({ allowClear: true });
```

Comment:
This will affect all `.select2` uses in the app – is this the only one, or are there some other that need checking?
(There's a 2i assignee select2 field if I remember correctly but I think that uses a different technique)